context/v1.jsonld

```diff
@@ -0,0 +1,71 @@
+{
+  "@context": {
+    "@protected": true,
+    "id": "@id",
+    "type": "@type",
+
+    "alsoKnownAs": {
+      "@id": "https://w3id.org/security#alsoKnownAs",
+      "@type": "@id",
+      "@container": "@set"
+    },
+    "assertionMethod": {
+      "@id": "https://w3id.org/security#assertionMethod",
+      "@type": "@id",
+      "@container": "@set"
+    },
+    "authentication": {
+      "@id": "https://w3id.org/security#authenticationMethod",
+      "@type": "@id",
+      "@container": "@set"
+    },
+    "capabilityDelegation": {
+      "@id": "https://w3id.org/security#capabilityDelegationMethod",
+      "@type": "@id",
+      "@container": "@set"
+    },
+    "capabilityInvocation": {
+      "@id": "https://w3id.org/security#capabilityInvocationMethod",
+      "@type": "@id",
+      "@container": "@set"
+    },
+    "controller": {
+      "@id": "https://w3id.org/security#controller",
+      "@type": "@id"
+    },
+    "keyAgreement": {
+      "@id": "https://w3id.org/security#keyAgreementMethod",
+      "@type": "@id",
+      "@container": "@set"
+    },
+    "verificationMethod": {
+      "@id": "https://w3id.org/security#verificationMethod",
+      "@type": "@id",
+      "@container": "@set"
+    },
+    "Multikey": {
+      "@id": "https://w3id.org/security#Multikey",
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "publicKeyMultibase": {
+          "@id": "https://w3id.org/security#publicKeyMultibase",
+          "@type": "https://w3id.org/security#multibase"
+        }
+      }
+    },
```

Comment:
```suggestion
   "Multikey": {
      "@id": "https://w3id.org/security#Multikey",
      "@context": {
        "@protected": true,
        "id": "@id",
        "type": "@type",
        "controller": {
          "@id": "https://w3id.org/security#controller",
          "@type": "@id"
        },
        "revoked": {
          "@id": "https://w3id.org/security#revoked",
          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
        },
        "expires": {
          "@id": "https://w3id.org/security#expiration",
          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
        },
        "publicKeyMultibase": {
          "@id": "https://w3id.org/security#publicKeyMultibase",
          "@type": "https://w3id.org/security#multibase"
        },
        "secretKeyMultibase": {
          "@id": "https://w3id.org/security#secretKeyMultibase",
          "@type": "https://w3id.org/security#multibase"
        }
      }
    },
```

Reply:
@dlongley, fine with me, but let me be awfully picky on one point. The `expires` property is defined in the DI specification in [2.1 Proofs](https://www.w3.org/TR/vc-data-integrity/#proofs). The definition says:

> The expires property is OPTIONAL and, if present, specifies when the proof expires.

There is no definition of the property in the controller spec. If I look at that definition, then it is undefined for a JWK or Multikey.

The way out would be to define the term more generally, and then specialize the term for proofs and for keys in the spec.

Reply:
Thinking about it further, and being aware of the importance of the `expires` property, one way forward could be to add a definition for `expires` into the section on [Verification Methods](https://www.w3.org/TR/controller-document/#verification-methods), with a similar definition as for proofs. However, that requires some changes on the vocabulary as well:

- The domain of the property should become a union of `Proof` and `VerificationMethod`
- At the moment, for all other terms the `definedBy` refers to one place only, i.e., the term is defined at one place. I would have to modify the script to allow for more to allow for more, and this property would then be defined at two different places. (Which should be all right.)

If we agree with this line, I am happy to do the work, but I would prefer for that to be the subject of a separate PR, and have this PR go along without that issue settled. The context files would then be modified along the lines of @dlongley's proposal.

Cc @msporny @selfissued

Reply:
Actually, I realized that, in the vocabulary, the term is already set with a union domain, so the fact that it does not appear in the controller document is just a bug. 

Having two formal definitions in the vocabulary should still be settled.